docs/getting_started/installation.md

```diff
@@ -581,10 +581,7 @@ Now that the system is built, you're going to need to install a few more depende
 
 Use Python 3's `pip` to install the following packages (this step will require **root** privileges):
 
-- jsonschema
-- psutil
 - pyodbc
-- requests
```

Comment:
- Update dependencies in cmake file for irods/irods
- Confirm the server does not need these

Reply:
psutil and jsonschema are used in testing. Will restore those.

I don't see any references to the requests module in tip of main. I grepped for `import\s*requests` and found nothing.

However, that same grep command was used against 4-3-stable and it did find one instance.

Reply:
Restored jsonschema and psutil.

Reply:
Confirmed core tests for irods/irods pass without the python 3 requests dependency.